recipes/recipes_emscripten/cairo/build.sh

```diff
@@ -3,6 +3,8 @@
 set -ex
 
 export CFLAGS="${CFLAGS} -DCAIRO_NO_MUTEX=1"
+export LDFLAGS="$(LDFLAGS) -sUSE_FREETYPE=1 -sUSE_PTHREADS=0"
+PTHREAD_CFLAGS=" "
```

Comment:
https://gitlab.freedesktop.org/cairo/cairo/-/blob/master/meson.build#L715

We may want to pass an option on top of it to set `CAIRO_HAS_PTHREAD` to 0.

Reply:
Done.